docs/guide/introduction.rst

```diff
@@ -13,24 +13,24 @@ Workflow overview
 
 The overall workflow of OpenFE involves three stages:
 
-1. **Setup**: Defining the simulation campaign you are going to run.
-2. **Execution**: Running and performing initial analysis of your
+1. :ref:`Simulation setup <userguide_setup>`: Defining the simulation campaign you are going to run.
+2. :ref:`Execution<userguide_execution>`: Running and performing initial analysis of your
    simulation campaign.
-3. **Gather results**: Assembling the results from the simulation
+3. :ref:`Gather results<userguide_results>`: Assembling the results from the simulation
    campaign for further analysis.
 
-In many use cases, these stages may be done on different machines -- for
+In many use cases, these stages may be done on different machines. For
 example, you are likely to make use of HPC or cloud computing resources to
-run the simulation campaign. Because of this, each stage has a certain type
-of output, which is the input to the next stage.
+run the simulation campaign. Because of this, each stage has a defined output which 
+is then the input for the next stage:
 
 .. TODO make figure
 .. .. figure:: ???
     :alt: Setup -> (AlchemicalNetwork) -> Execution -> (ProtocolResults) -> Gather
 
     The main stages of a free energy calculation in OpenFE, and the intermediates between them.
 
-The output of **setup** is an :class:`.AlchemicalNetwork`. This contains all
+The output of the **simulation setup** stage is an :class:`.AlchemicalNetwork`. This contains all
 the information about what is being simulated (e.g., what ligands, host proteins, solvation details etc) and the
 information about how to perform the simulation (the Protocol).
```

Comment:
So there's a somewhat intentional decision in the docs to avoid talking about `gufe`. The idea (and it is up for discussion/change) was that gufe was an abstract class that was non-user facing whilst `openfe` was the "user facing" library. I.e. object references would point to `openfe` not `gufe`.